docs/application/dotnet/guides/multimedia/thumbnail-images.md

```diff
@@ -27,17 +27,43 @@ using Tizen.Multimedia.Util;
 > **Note**   
 > The input media file can be common content in the device storage (internal or external) or private content in your application data.
 
-<a name="get_thumbnail"></a>
+
 ## Extracting a Thumbnail
 
 To extract a thumbnail from a file, use the `ExtractAsync()` method of the [Tizen.Multimedia.Util.ThumbnailExtractor](https://samsung.github.io/TizenFX/latest/api/Tizen.Multimedia.Util.ThumbnailExtractor.html) class:
 
 ```
-async Task Extract(string testImagePath)
+async Task ExtractAsync(string testImagePath, Size size)
+{
+    ThumbnailExtractionResult result = await ThumbnailExtractor.ExtractAsync(testImagePath, size);
+
+    Log.Info("Util", "Data size is " + result.RawData.Length);
+}
+```
+
+## Extracting a Thumbnail synchronously
+
+To extract synchronously a thumbnail from a file, use the `Extract(string path, Size size)` or `Extract(string path)` method of the [Tizen.Multimedia.Util.ThumbnailExtractor](https://samsung.github.io/TizenFX/latest/api/Tizen.Multimedia.Util.ThumbnailExtractor.html) class:
```

Comment:
@hsgwon ,request you to make a slight change.

Current:
To extract synchronously a thumbnail from a file, use the `Extract(string path, Size size)` or `Extract(string path)` method of the [Tizen.Multimedia.Util.ThumbnailExtractor](https://samsung.github.io/TizenFX/latest/api/Tizen.Multimedia.Util.ThumbnailExtractor.html) class:

Change to: 

To extract a thumbnail synchronously from a file, use the `Extract(string path, Size size)` or `Extract(string path)` method of the [Tizen.Multimedia.Util.ThumbnailExtractor](https://samsung.github.io/TizenFX/latest/api/Tizen.Multimedia.Util.ThumbnailExtractor.html) class:

Reason: Change the order of words to make the sentence grammatically appropriate:

Reply:
I'll fix it.